### PR TITLE
#498 Send license number and instructions via e-mail

### PR DESCRIPTION
**Pull request by jo-elimuai**

### CODE REVIEW

File: src/main/java/ai/elimu/web/project/license/LicenseCreateController.java [hunk 1]
```diff
@@ -93,6 +94,30 @@ public String handleSubmit(
             return "project/license/create";
         } else {
             licenseDao.create(license);
+            
+            // Send license information via e-mail
```
Comment (aschworer) [inline src/main/java/ai/elimu/web/project/license/LicenseCreateController.java]: I would put the whole thing into a separate method. It looks big!

Comment (aschworer) [reply src/main/java/ai/elimu/web/project/license/LicenseCreateController.java]: Did you think of any options to put HTML code on UI side?

Comment (jo-elimuai) [reply src/main/java/ai/elimu/web/project/license/LicenseCreateController.java]: > I would put the whole thing into a separate method. It looks big!

Yes, good idea. Will fix this.

Comment (jo-elimuai) [reply src/main/java/ai/elimu/web/project/license/LicenseCreateController.java]: > Did you think of any options to put HTML code on UI side?

What do you mean by UI side?

Comment (aschworer) [reply src/main/java/ai/elimu/web/project/license/LicenseCreateController.java]: > What do you mean by UI side?

I mean JSP. Normally it's a bad practice to include html into any Java code.
But in any case, this is an optional future refactoring

**Pull request merged**